data/registry/application-integration-java-keycloak.yml

```
@@ -0,0 +1,21 @@
+# cSpell:ignore keycloak
+title: Keycloak
+registryType: application integration
+language: java
+tags:
+  - keycloak
+  - iam
```

Possibly we could add one more tag specific to Keycloak?
```suggestion
tags:
  - keycloak
  - iam
  - sso
```